docs/source/_json/users.json

```diff
@@ -0,0 +1,26 @@
+GET /plone/@users
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+[
```

Comment:
@lukasgraf I added the @id attribute to the responses. I did not want to implement batching because I'm aware of the problem with get users you mentioned below. My idea was to push a basic implementation and then discuss the details. I should have made that more clear. :)